src/rocksdb/src/tuning.rs

```diff
@@ -241,7 +242,7 @@ impl FromStr for CompactionStyle {
     type Err = anyhow::Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
+        match UncasedStr::new(s).as_str() {
```

Comment:
This doesn't actually make the `match` uncased, you have to go through `UncasedStr`'s `PartialEq`/etc impls themselves, `as_str` just returns the original str

Reply:
Oh derp--ty